built_value_generator/lib/src/serializer_source_class.dart

```diff
@@ -270,7 +270,7 @@ class _\$${name}Serializer implements StructuredSerializer<$name> {
   final String wireName = '${escapeString(wireName)}';
 
   @override
-  Iterable serialize(Serializers serializers, $name object,
+  Iterable<Object> serialize(Serializers serializers, $name object,
```

Comment:
Er... did I do this?

Reply:
Apparently. I think you're going to need that for the generated code to be raw type safe?

Reply:
Haha yeah that's true. I'm just not sure how my change triggered it. But works for me!